HFE.md

```diff
@@ -0,0 +1,632 @@
+# Hash Items TTL Support in Valkey RFC
```

Comment:
There is a format, it would be nice if you could translate it https://github.com/valkey-io/valkey-rfc/blob/main/TEMPLATE.md.

Reply:
Yeh. I drifted from it. I can try and refactor it to match the format

Reply:
@madolson I made small refactoring to place the content better in the template format